packages/mobile/src/components/audio/AudioPlayer.tsx

```diff
@@ -441,20 +430,14 @@ export const AudioPlayer = () => {
 
     if (event.type === Event.PlaybackError) {
       console.error(`TrackPlayer Playback Error:`, event)
-      // Special recoverable case where the player was using a prefetched url but it failed
-      if (isUsingPrefetchUrl && track) {
-        // Unset the broken stream url, we don't want to accidentally use it again
-        dispatch(setStreamUrls({ [track.track_id]: undefined }))
-        // Get an updated track without a prefetched url
-        const updatedTrack = await makeTrackData(
-          {
-            track,
-            playerBehavior
-          },
-          true
-        )
-        TrackPlayer.load(updatedTrack)
-      }
+      const updatedTrack = await makeTrackData(
+        {
+          track,
+          playerBehavior
+        },
+        retries + 1
```

Comment:
Just confirming that a temporary client side network hiccup will cause us to bounce through the mirrors and not retry any of them more than once?
And did we do retries on the same URL before in this case? Just trying to understand what edge cases we're introducing by giving up on the first URL after one try.

Reply:
previously, a node would be chosen based on rendezvous hash and a byte range request for the first byte, using all nodes. Now, we take just the top N rendezvous hash and try streaming them until they work (or error). Notably, CN knows to redirect you to a node that does have the content if it does not have it. This means that despite trying all nodes before, you'd inevitably be stuck if the 4 nodes that were actually replicators didn't have the track despite trying all other nodes. The difference of behavior then is that the mirror count here is 2 so we only try 3 nodes instead of 4.

Reply:
(tl;dr no we haven't had retries before on the same node)